src/documents/classifier.py

```diff
@@ -28,16 +29,16 @@
 
 class IncompatibleClassifierVersionError(Exception):
     def __init__(self, message: str, *args: object) -> None:
-        self.message = message
+        self.message: str = message
         super().__init__(*args)
 
 
 class ClassifierModelCorruptError(Exception):
     pass
 
 
-def load_classifier() -> Optional["DocumentClassifier"]:
-    if not os.path.isfile(settings.MODEL_FILE):
+def load_classifier(raise_exception: bool = False) -> Optional["DocumentClassifier"]:
```

Comment:
Why the addition of `raise_exception`?

Reply:
That was my suggestion, if you look back at the failing version before I realized the try catch in the status view won’t actually catch anything because the inner part wouldn’t raise anything. Obviously if you have a different approach / solution or whatever feel free!

Reply:
Specifically https://github.com/paperless-ngx/paperless-ngx/pull/8325/commits/4212f6ebede387e010649702f23bedc96c7861ae + https://github.com/paperless-ngx/paperless-ngx/pull/8325/commits/1a942aab25a663eeaf7179db6e0c461b6b563b9f

Reply:
Ok, only request would be to make it keyword only for clarity.  Not sure why I haven't enabled [FBT002](https://docs.astral.sh/ruff/rules/boolean-default-value-positional-argument/#boolean-default-value-positional-argument-fbt002)

Reply:
Might as well be all the FBT.  I think I did just some rules, but maybe there are others which can help keep things clean.